server/config.ts

```diff
@@ -177,8 +177,19 @@ export default {
   ),
   featureToggles: {
     // someToggleEnabled: toBoolean(get('SOME_TOGGLE_ENABLED', false)),
-    reviewsPrisonsEnabled: get('REVIEWS_PRISONS_ENABLED', ''),
     completedGoalsEnabled: toBoolean(get('COMPLETED_GOALS_ENABLED', false)),
     archiveGoalNotesEnabled: toBoolean(get('ARCHIVE_GOAL_NOTES_ENABLED', false)),
+    reviewJourneyEnabledForPrison: (prisonId: string): boolean => {
+      const reviewsPrisonsEnabled = get('REVIEWS_PRISONS_ENABLED', '')
+        .split(',')
+        .map(id => id.trim())
+      return reviewsPrisonsEnabled.includes(prisonId) || reviewsPrisonsEnabled.includes('***')
+    },
+    prisonIsEnabledForService: (prisonId: string): boolean => {
```

Comment:
This function is the new feature toggle function property, and can be called to determine whether a given prison is one of the enabled prisons or not.